templates/global/product-tab.php

```diff
@@ -38,10 +38,11 @@
             </span>
         </li>
     <?php } ?>
-
+    <?php if ( class_exists( 'Dokan_Pro' ) || ( 'yes' === get_option( 'woocommerce_enable_reviews' ) ) ) : ?>
```

Comment:
`dokan()->is_pro_exists()` is preferred instead of `class_exists( 'Dokan_Pro' )`